druid.md

```diff
@@ -24,19 +24,17 @@ provides a framework for plugging in new functionality. Druid also supports
 approximate algorithms for cardinality estimation, and histogram and quantile
 calculations.
 
-**Interactive Queries** Druid’s query latency is optimized by reading and
-scanning only exactly what is needed. Aggregate and filter on data without
-sitting around waiting for results. Druid is ideal for powering analytic
-dashboards. Druid is a column-oriented data store and supports ad-hoc,
-  multi-dimensional filtering.
+**Sub-second Queries** Druid’s column orientation and inverted indexes enable
+scanning exactly what is needed for a query. Aggregate and filter on data in
+milliseconds. Druid is ideal for powering analytic dashboards. 
 
-**Real-time Data** Typical analytics databases ingest data via batches.
+**Real-time Ingestion** Typical analytics databases ingest data via batches.
 Ingesting an event at a time is often accompanied with transactional locks and
-other overhead that slows down the ingestion rate. Druid's real-time nodes
-employ lock-free ingestion of append-heavy data sets to allow for simultaneous
-ingestion and querying of 10,000+ events per second. Simply put, the latency
-between when an event happens and when it is visible is limited only by how
-quickly the event can be delivered to Druid.
+other overhead that slows down the ingestion rate. Druid employs lock-free
+ingestion of append-heavy data sets to allow for simultaneous ingestion and
+querying of 10,000+ events per second. Simply put, the latency between when an
```

Comment:
Per second per node? Per second per core? Otherwise this sounds like Druid can only handle 10,000 events per second